Python/assemble.c

```diff
@@ -482,28 +482,62 @@ extern void _Py_set_localsplus_info(int, PyObject *, unsigned char,
 
 static int
 compute_localsplus_info(_PyCompile_CodeUnitMetadata *umd, int nlocalsplus,
-                        PyObject *names, PyObject *kinds)
+                        int flags, PyObject *names, PyObject *kinds)
 {
+    // Compute the arg flags.
+    // Currently arg vars fill the first portion of the list.
+    char *argkinds = PyMem_Calloc(nlocalsplus, sizeof(char));
+    if (argkinds == NULL) {
+        PyErr_NoMemory();
+        return ERROR;
+    }
+    int numargvars = 0;
```

Comment:
Would it be possible to move this new code to after `kinds` was calculated, and then you don't need to allocate the `argkinds` array (just modify the `kinds` array)?

Reply:
I think I can make that work without slowing things down.

Reply:
done